src/Parser/Core/Modules/Items/BFA/ZandalariLoaFigurine.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+import SPELLS from "common/SPELLS";
+import ITEMS from "common/ITEMS";
+import Analyzer from "Parser/Core/Analyzer";
+import Combatants from "Parser/Core/Modules/Combatants";
+import { formatPercentage } from 'common/format';
+
+/**
+ * Zandalari Loa Figurine -
+ * Equip: Your spells and abilities have a chance to increase your primary stat by 814 for 10 sec.
+ */
+class ZandalariLoaFigurine extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  applications = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTrinket(
```

Comment:
While code style won't be a reason to request changes, I wish to point it out regardless for future reference. This should be a one liner; splitting it up in 3 lines makes it harder to read.